src/containers/government/GovernmentPage.js

```diff
@@ -0,0 +1,152 @@
+/*
+ * @flow
+ */
+
+import React from 'react';
+
+import { Colors } from 'lattice-ui-kit';
+
+import {
+  COLLABORATE,
+  CONNECT_INTO_PLATFORM,
+  COST_EFFECTIVE,
+  DISCOVER,
+  GET_COMPLETE_INFO,
+  GET_TIMELY_INFO,
+  HAVE_A_GOAL,
+  MODERN_PLATFORM,
+  RECEIVE_NOTIFICATIONS,
+  SEE_SHARE_INFO,
+  SHARE_INFO_SECURELY,
+  WHAT_DATA,
+  WHAT_UP_TO_DATE_INFO,
+  WHERE_YOU_LOOKING,
+  WHO_NEEDS_INFO,
+} from './constants/language';
+import { QUOTES } from './constants/quotes';
+
+import FooterSection from '../footer/FooterSection';
+import {
+  CostChartGraphic,
+  JaneSmithDataGraphic,
+  NotificationsGraphic,
+  PermissionsGraphic,
+  PlatformStackGraphic,
+} from '../../assets/svg/graphics';
+import {
+  FeatureGrid,
+  GraphicShot,
+  Header,
+  InfoTile,
+  PageSection,
+  QuotesSection,
+  ScrollToTop,
+  SectionContent,
+  SolidButton,
+  SubHeader,
+} from '../../components';
+import { openBeacon } from '../../utils/Utils';
+
+const { NEUTRAL } = Colors;
+
+const PAGE_SUB_HEADER = 'The OpenLattice data integration platform gives jurisdictions the flexibility to work with'
+  + ' any systems, departments, jurisdictions, and even researchers of choice, and full control over how to use data'
+  + ' to achieve goals.';
+
+const GovernmentPage = () => (
+  <>
+    <ScrollToTop />
+    <PageSection bgColor={NEUTRAL.N50}>
+      <SectionContent margin="20vh 0" maxWidth={{ sm: 450, lg: 700 }}>
+        <Header>Modern. Safe. Flexible.</Header>
+        <SubHeader>{PAGE_SUB_HEADER}</SubHeader>
+        <SolidButton onClick={openBeacon}>Chat with us</SolidButton>
+      </SectionContent>
+    </PageSection>
+    <PageSection>
+      <SectionContent>
+        <FeatureGrid reverseOrderOnWrap>
+          <GraphicShot>
+            <PlatformStackGraphic />
+          </GraphicShot>
+          <InfoTile align="left">
+            <Header as="h2" color={NEUTRAL.N800}>{MODERN_PLATFORM}</Header>
+            <SubHeader as="h5" color={NEUTRAL.N700}>{CONNECT_INTO_PLATFORM}</SubHeader>
+            <SubHeader as="h5" color={NEUTRAL.N700}>{HAVE_A_GOAL}</SubHeader>
```

Comment:
we probably should not be skipping heading levels like this for accessibility reasons.
https://www.w3.org/WAI/tutorials/page-structure/headings/#heading-ranks

Material gets around this by using `component` and `variant` props for `<Typography />`. if the designs don't match the typography from the component library (though I would hope so) I would suggest adjusting the implementation of Header and SubHeader to mimic this behavior of decoupling html semantics from styles.

Also adding appropriate id attributes would be helpful to navigate between sections.

Reply:
Great points. I'll switch to LUK `Typography` component in a follow-up pr.